### ISSUE TITLE
[Toolkit bug]: 

### ISSUE THREAD

**Issue by PapaSlask** [NVIDIAGameWorks/rtx-remix#362]
### Describe the bug

Don't know if bug or a missing feature but... I would love to click on my 3D asset and access the mesh .usd file and its structure and properties. Without accessing the Xform I can't safely and accurately move my model around. A workaround for now is to find the target "parent" mesh, click on it, and find my .usd mesh in the list. Here's two sceenshots to show what happens when clicking on the imported/replaced .usd mesh, and when going via the "parent" 
Imported mesh:
![111](https://github.com/NVIDIAGameWorks/rtx-remix/assets/11363281/b76ce1c7-8c2b-4f39-b45c-d742cba473a2)

Parent:
![222](https://github.com/NVIDIAGameWorks/rtx-remix/assets/11363281/9233423b-6dea-4e59-a800-a451310bba3b)

### How do you reproduce the bug?

As described above

### What is the expected behavior?

The toolkit should...

### Version

2024.1.1

**Comment by nvdamien** [comment]
Hey @PapaSlask  ,

The 2 images that you are showing are 2 different assets (2 different hash: A3A...., 062...).
There is no hierarchy on asset, all assets (mesh_XXXXXX) are in a flat list, same level.

From here, can you give us more detail about:
> A workaround for now is to find the target "parent" mesh

Please?

Also, if I understand, the problem is a selection problem?

**Comment by PapaSlask** [comment]
> Hey @PapaSlask ,
> 
> The 2 images that you are showing are 2 different assets (2 different hash: A3A...., 062...). There is no hierarchy on asset, all assets (mesh_XXXXXX) are in a flat list, same level.
> 
> From here, can you give us more detail about:
> 
> > A workaround for now is to find the target "parent" mesh
> 
> Please?
> 
> Also, if I understand, the problem is a selection problem?

Hi Damien! Yes the first image has a different hash because I have the metal pillar stuck in selection, it remained even after I selected my custom 3d mesh again, it didn't get deselected. I understand how that added an extra layer of confusion, sorry about that. But to explain the situation better I have prepared a video of the phenomenon: 

https://github.com/NVIDIAGameWorks/rtx-remix/assets/11363281/c67905ba-e38c-4133-8824-3399263e1f8b

I would like to get the model, the parent, the hierarchy structure, everything, when I click on my imported 3D mesh. For now I have to click on its "parent" to get there.

**Comment by PapaSlask** [comment]
An update on this! I just ingested and imported a proper fbx model. When I select it in the 3d viewport, I get all the stuff I asked for. This seem to be a problem when ingesting .usd models. Please investigate!

**Comment by nvdamien** [comment]
Hey @PapaSlask ,

From your video:

![image](https://github.com/NVIDIAGameWorks/rtx-remix/assets/128181406/09bfacc0-66ae-45dc-9392-713d6eda2274)

This is not normal to have a mesh from a capture that has no instance, no mesh prim.

Did you delete anything from the capture layer by hand, or outside of the RTX Remix app? Or did you add an asset "as a layer" (and not as a replacement)?

**Comment by PappaSlask** [comment]
> Hey @PapaSlask ,
> 
> From your video:
> 
> ![image](https://private-user-images.githubusercontent.com/128181406/300570411-09bfacc0-66ae-45dc-9392-713d6eda2274.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4MTQ3NzUsIm5iZiI6MTcwNjgxNDQ3NSwicGF0aCI6Ii8xMjgxODE0MDYvMzAwNTcwNDExLTA5YmZhY2MwLTY2YWUtNDVkYy05MzkyLTcxM2Q2ZWRhMjI3NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwMVQxOTA3NTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNTgxZDA2ODFjMTM1MWRhYmU0NDc0OTVkYjEzY2IxMDM1NDJlNWNiMjIzMmI1ODI5ODMxYmZmZTQ3NTRiOTIwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Yxwh7TvF3Z7t9dlHhXLwORuf1s2Fr6i2maNxL4iLpww)
> 
> This is not normal to have a mesh from a capture that has no instance, no mesh prim.
> 
> Did you delete anything from the capture layer by hand, or outside of the RTX Remix app? Or did you add an asset "as a layer" (and not as a replacement)?

Hello! It's not a capture layer, it's a 3d asset that I exported from Composer. I exported it as .usd. And it has no Xforms? I don't have enough experience to fully understand this.

**Comment by PappaSlask** [comment]
> Did you ingest the asset?

Yes I did, that's what's so weird. I will investigate this further today and report back!

**Comment by sjgillard** [comment]
Closing this issue. There have been many selection improvements since this was filed. Open a new issue if you are still having issues.
